packages/ui-components/src/lib/components/deployment/getDeploymentTransactionArgs.ts

```
@@ -11,30 +9,36 @@ export enum AddOrderErrors {
+	network: string;
 }
 
 export async function getDeploymentTransactionArgs(
 	gui: DotrainOrderGui,
-	wagmiConfig: Config | undefined
+	account: string | null
 ): Promise<HandleAddOrderResult> {
-	if (!wagmiConfig) {
-		throw new Error(AddOrderErrors.MISSING_CONFIG);
-	}
+	let network: string;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'network' is only assigned here.


(lint/style/useConst)